fairscale/nn/data_parallel/sharded_ddp.py

```diff
@@ -491,6 +454,10 @@ def _setup_backward_hooks(self) -> None:
         This makes the gradient reduction automatic whenever there's a backward pass
         """
 
+        # Empty the possible previous hooks
```

Comment:
suggested in the doc, doing that for correctness, I'm not sure of the effect. Remove the previous hooks before attaching new ones